pages/tips/index.js

```diff
@@ -1,24 +1,166 @@
+          <h3>Choose a category to get the corresponding tips & tricks:</h3>
+          <StyledForm>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Food"
+                id="essen"
+                value="Food"
+                checked={selectedCategories.includes("Food")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="essen">Food</StyledLabel>
+            </StyledDiv>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Clothes"
+                id="kleidung"
+                value="Clothes"
+                checked={selectedCategories.includes("Clothes")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="Kleidung">Clothes</StyledLabel>
+            </StyledDiv>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Household"
+                id="haushalt"
+                value="Household"
+                checked={selectedCategories.includes("Household")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="haushalt">Household</StyledLabel>
+            </StyledDiv>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Entertainment"
+                id="unterhaltung"
+                value="Entertainment"
+                checked={selectedCategories.includes("Entertainment")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="unterhaltung">Entertainment</StyledLabel>
+            </StyledDiv>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Gasoline"
+                id="benzin"
+                value="Gasoline"
+                checked={selectedCategories.includes("Gasoline")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="benzin">Gasoline</StyledLabel>
+            </StyledDiv>
+            <StyledDiv>
+              <StyledCheckboxInput
+                type="checkbox"
+                name="Restaurants"
+                id="restaurants"
+                value="Restaurants"
+                checked={selectedCategories.includes("Restaurants")}
+                onChange={handleToggleFilter}
+              />
+              <StyledLabel htmlFor="restaurants">Restaurants</StyledLabel>
+            </StyledDiv>
+          </StyledForm>
+          {filteredTips.map(filteredTip => {
+            return (
+              // eslint-disable-next-line react/jsx-key
+              <ListItem key={filteredTip.id}>
+                <Tip tips={tip} tip={filteredTip} />
+              </ListItem>
+            );
+          })}
+        </StyledBody>
+      </div>
+    </>
```

Comment:
I would suggest you the following:
- In this case its better to use only the fragment `<>` and no extra `<div>`
- Don't try to disable eslint.
- Do you really need an extra `div` around every `ínput`
- Better write the values of your inputs in lower case.

```suggestion
    <>
        <StyledHeader>
          <StyledTitle>Best Tips & Tricks</StyledTitle>
        </StyledHeader>
        <StyledBody>
          <h3>Choose a category to get the corresponding tips & tricks:</h3>
          <StyledForm>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Food"
                id="essen"
                value="food"
                checked={selectedCategories.includes("Food")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="essen">Food</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Clothes"
                id="kleidung"
                value="clothes"
                checked={selectedCategories.includes("Clothes")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="Kleidung">Clothes</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Household"
                id="haushalt"
                value="household"
                checked={selectedCategories.includes("Household")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="haushalt">Household</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Entertainment"
                id="unterhaltung"
                value="entertainment"
                checked={selectedCategories.includes("Entertainment")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="unterhaltung">Entertainment</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Gasoline"
                id="benzin"
                value="gasoline"
                checked={selectedCategories.includes("Gasoline")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="benzin">Gasoline</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Restaurants"
                id="restaurants"
                value="restaurants"
                checked={selectedCategories.includes("Restaurants")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="restaurants">Restaurants</StyledLabel>
            </StyledDiv>
          </StyledForm>
          {filteredTips.map(filteredTip => {
            return (
              <ListItem key={filteredTip.id}>
                <Tip tips={tip} tip={filteredTip} />
              </ListItem>
            );
          })}
        </StyledBody>
    </>
```

Reply:
I see, thanks @stefankorder ! I'm updating this PR according to your comments.